okio/build.gradle.kts

```diff
@@ -132,6 +132,11 @@ kotlin {
           nativeTest.dependsOn(nonJvmTest)
           nativeTest.dependsOn(nonWasmTest)
           createSourceSet("appleTest", parent = nativeTest, children = appleTargets)
+            .apply {
+              dependencies {
+                implementation(libs.kotlin.coroutines)
```

Comment:
Is it used only in tests? Should be `testImplementation` instead

Reply:
Yes, it is only used in tests. But this is specifically the `appleTest` source set `KotlinDependencyHandler` dependency management block. `testImplementation` is only valid in the top-level `dependencies` block, and applies specifically for the `test` source set, which isn't specific enough for a KMP module with multiple platform target `main` and `test` source sets.